OmgvaPOS/src/ItemManagement/Controller/ItemController.cs

```diff
@@ -123,39 +86,25 @@ public IActionResult CreateItem([FromBody] CreateItemRequest createItemRequest)
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status500InternalServerError)]
         public IActionResult UpdateItem([FromBody] ItemDTO item, long id) {
-            long businessId = _itemService.GetItemNoException(id).BusinessId;
-            if (!JwtTokenHandler.CanManageBusiness(HttpContext.Request.Headers.Authorization!, businessId))
+            long itemBusinessId = _itemService.GetItemNoException(id).BusinessId;
+            if (!AuthorizationHandler.CanManageBusiness(HttpContext.Request.Headers.Authorization!, _itemService.GetItemNoException(id).BusinessId))
                 return Forbid();
 
             item.Id = id;
-            item.Currency = item.Currency.ToUpper();
+            item.Currency = item.Currency?.ToUpper();
 
-            if (item.UserId != null && _userService.GetUser((long)item.UserId).BusinessId != businessId) //TODO: User id may be wrong here
+            if (item.UserId != null && _userService.GetUser((long)item.UserId).BusinessId != itemBusinessId) //TODO: User id may be wrong here
```

Comment:
I think this was just merged, take a good look at it. It is the same in create and update endpoints.

Reply:
Yes, this is the result of a merge conflict resolution